### ISSUE TITLE
I'd love a way to send code to a terminal with Cmd + Return

### ISSUE THREAD

**Issue by toph-allen** [jonclayden/R-Nova#1]
I haven't been able to figure out if this feature exists, and if so, how to enable it.

**Comment by jonclayden** [comment]
I agree this would be neat! To be clear, I assume you mean a terminal tab within Nova, rather than in Terminal.app or elsewhere? As far as I can tell, the Nova API doesn't currently provide a way to do this – and there would be some ambiguity to resolve if there were multiple terminals open – but I will look further into it. If you're aware of any other extensions that do something similar then do let me know.

**Comment by jonclayden** [comment]
The Nova developers have confirmed that this isn't currently possible, and opened an upstream feature request to add it. I will work on this feature once it's supported by the editor!

**Comment by toph-allen** [comment]
Awesome! Cool that they'll be supporting that in the future. :)
